tests/kafkatest/tests/core/downgrade_test.py

```diff
@@ -71,7 +76,16 @@ def setup_services(self, kafka_version, compression_types, security_protocol):
                              version=kafka_version)
         self.consumer.start()
 
+    def wait_until_rejoin(self):
+        for partition in range(0, self.PARTITIONS):
+            wait_until(lambda: len(self.kafka.isr_idx_list(self.topic, partition)) == self.REPLICATION_FACTOR, 
+                    timeout_sec=60, backoff_sec=1, err_msg="Replicas did not rejoin the ISR in a reasonable amount of time")
+
     @cluster(num_nodes=7)
+    @parametrize(version=str(LATEST_2_5), compression_types=["none"])
```

Comment:
These additions don't work because of the changes to allow for static group membership. I will revert them and we can address it in a follow up PR.